torch/_inductor/runtime/benchmarking.py

```diff
@@ -1,19 +1,45 @@
 import time
 from functools import cached_property
+from functools import cached_property, wraps
 from statistics import median
 from typing import Any, Callable, Dict, List, Tuple
 from typing_extensions import Self
 
+import torch
 from torch._inductor.utils import is_cpu_device
 
 
+log = torch._logging.getArtifactLogger(__name__, "benchmarking")
+
+
 MILLISECONDS_PER_SECOND = 1000
 
 
+def maybe_time(fn: Callable[..., Any]) -> Callable[..., Any]:
```

Comment:
nit: use mypy ParamSpec to type

Reply:
oops looks like this got removed in a merge, added again